x-pack/plugins/log_explorer/public/components/virtual_columns/resource.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import type { DataGridCellValueElementProps } from '@kbn/unified-data-table';
+import { first } from 'lodash';
+import { AgentName } from '@kbn/elastic-agent-utils';
+import { ChipWithPopover } from '../common/popover_chip';
+import * as constants from '../../../common/constants';
+import { getUnformattedResourceFields } from '../../utils/resource';
+import { LogDocument } from '../../../common/document';
+import { dynamic } from '../../utils/dynamic';
+
+const AgentIcon = dynamic(() => import('@kbn/custom-icons/src/components/agent_icon'));
```

Comment:
**question**: same as https://github.com/elastic/kibana/pull/175287/files#r1464995336

Reply:
This increased the bundle size from 48kb to 128kb